cpp/ql/src/Best Practices/Magic Constants/MagicConstantsNumbers.qhelp

```diff
@@ -20,7 +20,7 @@ update if the requirements change (for example, one more guest is permitted).
 </li>
 </ol>
 
-<p>This rule finds magic numbers for which there is no pre-existing named
+<p>This rule finds magic numbers for which there is no preexisting named
```

Comment:
I'm happy to drop any particular change(s). This is a word, and thus there's no particular reason to use the hyphenated flavor.